Unbreak //c10/util:base

swolchok opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #156216

Missing dep.

Differential Revision: [D76840057](https://our.internmc.facebook.com/intern/diff/D76840057/)

swolchok committed:
Unbreak //c10/util:base

Missing dep.

Differential Revision: [D76840057](https://our.internmc.facebook.com/intern/diff/D76840057/)

[ghstack-poisoned]

janeyx99 reviewed (commented):
@desertfire you had also removed this in caffe2/c10/core/build.bzl -- would that be needed here too?

swolchok committed:
Update on "Unbreak //c10/util:base"


Missing dep.

Differential Revision: [D76840057](https://our.internmc.facebook.com/intern/diff/D76840057/)

[ghstack-poisoned]

swolchok commented:
@janeyx99 you accepted internally; did you mean to accept here too?

swolchok commented:
@pytorchbot merge

swolchok committed:
Update on "Unbreak //c10/util:base"


Missing dep.

Differential Revision: [D76840057](https://our.internmc.facebook.com/intern/diff/D76840057/)

[ghstack-poisoned]

swolchok commented:
bypassing https://github.com/pytorch/pytorch/actions/runs/15738554902/job/44358937049?pr=156216 failure as it appears to e due to a missing file which is certainly not missing because of this PR . same for [trunk / macos-py3-arm64 / test (default, 1, 3, macos-m1-stable) (push)](https://github.com/pytorch/pytorch/actions/runs/15738556702/job/44358454145?pr=156216)
trunk / macos-py3-arm64 / test (default, 1, 3, macos-m1-stable) (push)

atalman commented:
@pytorchmergebot merge -f "already merged internally"
